### ISSUE TITLE
[Feature Request] Allow alternative action names

### ISSUE THREAD

**Issue by tillsanders** [Cloudstek/alfred-hugo#7]
I think it would be helpful to be able to provide alternative names for actions. The `Hugo.action()` method could accept a string or an array of strings, like this:

```javascript
hugo.action('context', () => {})
hugo.action(['c', 'ctx', 'context'], () => {})
```
 All these keywords would be mapped to the same function. Of course, this would only be a bit of syntactic sugar.

**Comment by mdeboer** [comment]
Ah cool yeah, aliases! I will add this for v2.1.0 👍

**Comment by mdeboer** [comment]
This has been released in v3.0.0 and is available now! 👍
